src/main/java/org/datadog/jmxfetch/JmxTabularAttribute.java

```diff
@@ -133,10 +135,12 @@ private void populateSubAttributeList(Object value) {
             String dataKey = entry.getKey();
             List<String> subSub = entry.getValue();
             for (String metricKey : subSub) {
-                String alias = convertMetricName(getAlias(metricKey));
+                String alias = getAlias(metricKey);
```

Comment:
🍰 this should be fine because `getAlias()` already calls `convertMetricName`, we were just duplicating work.